registration/src/main/java/com/kyriba/registration/api/dto/AttendeeTransition.java

```diff
@@ -0,0 +1,17 @@
+package com.kyriba.registration.api.dto;
+
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+@Data
+@NoArgsConstructor
+public class AttendeeTransition
```

Comment:
Why *Transition?

Reply:
Just thinking what name is better. Should it be "AttendeeRequest"?

Reply:
Traditional naming would be AttendeeRegistrationRequest.

Reply:
done